src/App.js

```diff
@@ -3,17 +3,19 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 import About from './components/support';
 import Home from './components/home';
 
-/*
-The principal component for housing sub-components (like home, about and
+
+
+/*The principal component for housing sub-components (like home, about and
 other pages) for easy navigation
 */
+
 class App extends Component {
     render() {
         return (
             <Router>
                 <div className="App">
                    <Route path='/' exact strict component={Home} />
-                   <Route path='/about/' exact strict component={About} />
+                   <Route path='/about/' exact strict component={About} />               
```

Comment:
delete the extra space at the end of this line

Reply:
Done